exasol_integration_test_docker_environment/doctor.py

```diff
@@ -79,13 +84,12 @@ def health_checkup() -> Iterable[error.ExaError]:
 
     return an iterator of error codes specifying which problems have been identified.
     """
-    check_function = Callable[[],bool]
-    diagnosis_function = Callable[[],Iterable[error.ExaError]]
-    examinations : List[Tuple[check_function, diagnosis_function]] = [
+    check_function = Callable[[], bool]
+    diagnosis_function = Callable[[], Iterable[error.ExaError]]
+    examinations: List[Tuple[check_function, diagnosis_function]] = [
         (is_docker_daemon_available, diagnose_docker_daemon_not_available),
         (is_supported_platform, lambda: [Error.TargetPlatformNotSupported]),
     ]
     for is_fine, diagnosis in examinations:
         if not is_fine():
-            for error_code in diagnosis():
-                yield error_code
+            yield from diagnosis()
```

Comment:
@Nicoretti scary that `nox -s fix` does this kind of change. But I believe it's correct

Reply:
`pyupgrade` actually pyupgrade does this. If you prefer syntax only we can discuss and introduce new/other targets.

Reply:
Additionally imho this is a clear improvement ;P